src/smtsolvers/GhostSMTSolver.cc

```diff
@@ -188,11 +188,9 @@ GhostSMTSolver::pickBranchLit() {
 }
 
 void GhostSMTSolver::relocAll() {
-    for (int i = 0; i < thLitToClauses.size(); i++) {
-        vec<CRef> &appearances = thLitToClauses[i];
-        for (int j = 0; j < appearances.size(); j++) {
-            CRef cr = appearances[j];
-            appearances[j] = ca[cr].relocation();
+    for (auto & appearances : thLitToClauses) {
+        for (CRef & cr : appearances) {
```

Comment:
`auto const & appearances` and `CRef cr`?

Reply:
Fixed.